catalog/raw/redis/redis-7.0.10.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: catalog.kubedb.com/v1alpha1
+kind: RedisVersion
+metadata:
+  name: 7.0.10
+spec:
+  coordinator:
+    image: kubedb/redis-coordinator:v0.9.0
+  db:
+    image: redis:7.0.10
+  exporter:
+    image: kubedb/redis_exporter:1.9.0
```

Comment:
All kubedb images must be from `ghcr.io/kubedb`

Reply:
Updated